mmv1/products/firebase/terraform.yaml

```diff
@@ -57,6 +57,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           - project
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/self_link_as_name.erb
+  AppleApp: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ['{{project}} {{name}}']
+    autogen_async: true
+    skip_delete: true #currently only able to delete a appleapp through the Firebase Admin console
+    skip_sweeper: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "firebase_apple_app_basic"
+        min_version: "beta"
+        primary_resource_id: "basic"
+        vars:
+          display_name: "Display Name Basic"
+          app_store_id: "App Store ID Basic"
+          team_id: "Team ID Basic"
+        test_env_vars:
```

Comment:
we might be better off using test_env_overrides for the IDs to use semi-realistic values
ex:   
```bundle_id: "apple.app." + randString(t, 5)" ```